index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
       
     </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <div class="card__image">
+          <img src="images/tox1.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
```

Comment:
- [ ] Speaking of specifying paths for images, here is another bug that is preventing your beautiful images to be displayed on your page 🤔. To fix this kindly capitalize the first letter of the `images` directory, so you end up with something like this `src="Images/tox1.png"`

![image](https://user-images.githubusercontent.com/43172164/206504323-337a4114-f0bf-4a49-b6be-1218b6753a0c.png)